src/main/java/server/response/HeaderHelpers.java

```diff
@@ -0,0 +1,44 @@
+package server.response;
+
+import server.request.Handler;
+import server.response.stringcomponents.ContentTypes;
+import server.response.stringcomponents.HTTP;
+
+import java.io.File;
+import java.util.Map;
+import java.util.Set;
+
+public class HeaderHelpers {
+    private HeaderHelpers() {
+    }
+
+    public static String allowedMethods(Map<String, Handler> route) {
+        if(route == null)
+            return "GET, HEAD";
+
+        Set<String> methodSet = route.keySet();
+        String[] allowedMethods = methodSet.toArray(new String[methodSet.size()]);
+        return String.join(", ", allowedMethods);
+    }
+
+    public static String location(String host, String path) {
+        return HTTP.URL_PREFIX + host + path;
+    }
+
+    public static String contentType(File file) {
+        String extension = fileExtension(file);
+        String contentType = ContentTypes.HEADER_VALUES.get(extension);
+        if(contentType == null)
+            return ContentTypes.HEADER_VALUES.get("");
+        return contentType;
+    }
+
+    private static String fileExtension(File file) {
+        String name = file.getName();
+        int lastIndexOf = name.lastIndexOf(".");
+        if (lastIndexOf == -1) {
```

Comment:
why brackets here and not above for the if statements?

Reply:
No good reason at all! I've removed them, thanks.